Linked_List/Linked_List.cpp

```diff
@@ -173,10 +173,9 @@ void Length_Iterative()
 
     if(!Is_List_Empty())
```

Comment:
@Sameer87 Thanks for pointing that out. I think that this 'if' statement is also extra here. Will you update it for us?